### ISSUE TITLE
`--passes` appears to be ignored

### ISSUE THREAD

**Issue by tsalo** [bbfrederick/rapidtide#178]
**Describe the bug**
I ran the 28andMe dataset through rapidtide with `--passes 50`, but the runoptions dict shows that `passes` is set to 3.

**To Reproduce**
```bash
rapidtide \
    /data/sub-01/ses-04/func/sub-01_ses-04_task-rest_space-MNI152NLin6Asym_res-2_desc-preproc_bold.nii.gz \
    /out/sub-01_ses-04_task-rest_space-MNI152NLin6Asym_res-2 \
    --delaymapping \
    --brainmask /data/sub-01/ses-04/func/sub-01_ses-04_task-rest_space-MNI152NLin6Asym_res-2_desc-brain_mask.nii.gz \
    --graymattermask /out/sub-01_ses-04_space-MNI152NLin6Asym_res-2_label-GM_mask.nii.gz \
    --whitemattermask /out/sub-01_ses-04_space-MNI152NLin6Asym_res-2_label-WM_mask.nii.gz \
    --motionfile /data/sub-01/ses-04/func/sub-01_ses-04_task-rest_desc-confounds_timeseries.tsv \
    --similaritymetric hybrid \
    --passes 50 \
    --despecklepasses 50 \
    --filterband lfo \
    --numtozero 3 \
    --datatstep 0.72 \
    --nprocs 10 \
    --noprogressbar
```

**Expected behavior**
The `--delaymapping` analysis type shouldn't override individual parameters, so I figured that `passes` would be set to 50.

**Desktop (please complete the following information):**
 - OS: Ubuntu, run with Apptainer
 - Version: v2.9.8+0.g71557e9c

**Comment by bbfrederick** [comment]
Yeah, that's definitely a bug!  I'll look at it in a bit.

**Comment by bbfrederick** [comment]
This is fixed in [v2.9.9.1](https://github.com/bbfrederick/rapidtide/releases/tag/v2.9.9.1).  I also put in a check to stop despeckling when it's not changing anything.
